apps/enmeshed/lib/account/my_data/initial_creation/my_data_initial_creation_screen.dart

```diff
@@ -176,7 +180,7 @@ class _MyDataInitialCreationScreenState extends State<MyDataInitialCreationScree
           _valueRendererValidationErrors.remove(valueType);
         }
 
-        _updateSaveEnabled();
+        if (mounted) setState(() => _saveEnabled = _attributeValues.isNotEmpty || _valueRendererValidationErrors.isNotEmpty);
```

Comment:
When checking the deleted `_updateSaveEnabled` method, I'd expect:

```suggestion
        if (mounted) setState(() => _saveEnabled = _attributeValues.isNotEmpty && _valueRendererValidationErrors.isEmpty);
```